src/unstract/sdk/audit.py

```diff
@@ -102,7 +102,9 @@ def push_usage_data(
                     level=LogLevel.ERROR,
                 )
             else:
-                self.stream_log("Successfully pushed usage details")
+                self.stream_log(
+                    "Successfully pushed usage details", level=LogLevel.DEBUG
```

Comment:
Will it help to log actual counts against the model name as DEBUG?

Reply:
@gaya3-zipstack do you mean displaying the actual usage that got pushed?

Reply:
Logging the entire usage dict here - hope that's okay